src/axom/inlet/tests/inlet_Inlet.cpp

```diff
@@ -1058,8 +1058,119 @@ TEST(inlet_Inlet_verify, verifyValidStringValues) {
   field = inlet1->addString("NewTable/field5");
   field->validValues(strs)->defaultValue("zyx");
   EXPECT_FALSE(inlet1->verify());
+}
+
+TEST(inlet_verify, verifyFieldLambda) {
+  std::string testString = "field1 = true; field2 = 'abc'; NewTable = { field3 = 'xyz'; field4 = 'yes' }";
+  DataStore ds;
+  auto inlet = createBasicInlet(&ds, testString);
+
+  auto field1 = inlet->addBool("field1");
+  auto field2 = inlet->addString("field2");
+  auto field3 = inlet->addString("NewTable/field3");
+
+  field2->registerVerifier([&]() -> bool {
+    std::string str = "";
+    inlet->get("field2", str);
+    return (str.size() >= 1 && str[0] == 'a');
+  });
+  EXPECT_TRUE(inlet->verify());
+
+  field3->registerVerifier([&]() -> bool {
+    std::string str = "";
+    inlet->get("NewTable/field3", str);
+    return (str.size() >= 1 && str[0] == 'a');
+  });
+  EXPECT_FALSE(inlet->verify());
+}
+
+TEST(inlet_verify, verifyTableLambda) {
+  std::string testString = "field1 = true; field2 = 'abc'; NewTable = { field3 = 'xyz'; field4 = 'yes' }";
+  DataStore ds;
+  auto inlet = createBasicInlet(&ds, testString);
+
+  auto field1 = inlet->addBool("field1");
+  auto field2 = inlet->addString("field2");
+  auto table1 = inlet->addTable("NewTable");
+  auto field3 = table1->addString("field3");
+
+  field2->registerVerifier([&]() -> bool {
+    std::string str = "";
+    inlet->get("field2", str);
+    return (str.size() >= 1 && str[0] == 'a');
+  });
+  EXPECT_TRUE(inlet->verify());
+
+  field3->registerVerifier([&]() -> bool {
+    std::string str = "";
+    inlet->get("NewTable/field3", str);
+    return (str.size() >= 1 && str[0] == 'x');
+  });
+  EXPECT_TRUE(inlet->verify());
+
+  table1->registerVerifier([&]() -> bool {
+    return table1->hasField("field3");
+  });
+  EXPECT_TRUE(inlet->verify());
```

Comment:
Could you please clarify something:  
What would have happened if you had `table1->required()`  and it was not present? I.e. which code check would fail and where and when would that happen?

Reply:
inlet->verify() will fail